daemon/pool.go

```diff
@@ -32,6 +36,31 @@ var (
 	errDriverStopped = errors.NewKind("driver stopped")
 )
 
+var (
+	policyDefaultWindow    = envIntOr("BBLFSHD_POLICY_WINDOW", 30)
+	policyDefaultMin       = envIntOr("BBLFSHD_POLICY_MIN", 1)
+	policyDefaultScale     = envIntOr("BBLFSHD_POLICY_SCALE", 1)
+	policyDefaultDownscale = envFloatOr("BBLFSHD_POLICY_DOWNSCALE", 0.25)
+)
+
+func envIntOr(env string, def int) int {
+	s := os.Getenv(env)
+	v, err := strconv.Atoi(s)
+	if err != nil {
+		return def
```

Comment:
Since this is done at startup, it is probably OK to panic/fatal for issues here. Or at least, to log the error rather than silently using the default. 

I'd probably make this:

```go
func mustEnvInt(env string, def int) int {
   s := os.Getenv(env)
   if s == "" {
      return def
   }
   v, err := strconv.Atoi(s)
   if err != nil {
      log.Fatalf("Invalid value for %q: %q", env, s)
   }
   return v
}
```

(the "must" prefix is similar to the C++ convention "…OrDie" in some codebases)